default/templates/default/mapping_sheet.html

```diff
@@ -16,18 +16,23 @@
                     names, paths and descriptions of all OCDS fields in a table format.
                 {% endblocktrans %}
             </p>
-            <p>{% trans "The options available are" %}:</p>
+            <p>{% trans "The options available are:" %}</p>
             <ul class="list-unstyled">
-                <li><strong>{% trans "Select a schema" %}:</strong> {% trans "Use any of the official OCDS Schemas" %}.
```

Comment:
Is there anything wrong with leaving the colon outside the translatable text? I did it this way in some instances to avoid duplication of these strings.

Reply:
Colons are translatable text, e.g. in Armenian a colon indicates the end of a sentence.

In general, all text and all punctuation is linguistic and should be in the string to translate.

Reply:
I see, thanks for clarifying.